src/components/components/ebay-dialog-base/index.marko

```diff
@@ -98,6 +101,9 @@ $ var header = input.header;
         <div id=input.mainId class=`${input.classPrefix}__main`  key="body" onScroll("handleScroll")>
             <${input.renderBody}/>
         </div>
+        <span class=`${input.classPrefix}__actions`>
+            <${input.action && input.action.renderBody}/>
+        </span>
```

Comment:
If no action is present then this section should not get rendered.
Can we add it as a part of main or does it have to be it's own section? 
I'm fine, but generally we had header, main, footer

Reply:
Okay putting it in the main section causes CSS weirdness. But I added an if tag so it doesn't render if there's no action.